folk_rnn_site/composer/templates/composer/_header.html

```diff
@@ -27,7 +27,9 @@ <h2 class="content-subhead">What is ABC Notation?</h2>
             <li><a href="http://abcnotation.com/blog/2010/01/31/how-to-understand-abc-the-basics/">Understanding ABC</a></li>
             <li><a href="http://abcnotation.com/wiki/abc:standard:v2.1">ABC format standard</a>
         </ul>
+        <h2 class="content-subhead">What's the difference between the models?</h2>
+        <p>In the controls on the left, you can select between two models built using data from <a href="http://thesession.org">thesession.org</a>: (w/ :||:) and (w/o  :||:). The first model is trained on abc with the repeat signs preserved. The second is trained on abc with the repeats made explicit.</p>
         <h2 class="content-subhead">Are there issues with playback on this site?</h2>
```

Comment:
Took me a moment to realise that 'made explicit' wasn't a mistake. The programmer term would be 'loops unrolled'. Is there a better example or clearer way of saying this?